### PR TITLE
MNT Travis shared config

### PR DESCRIPTION
**Pull request by emteknetnz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,106 +1,62 @@
...
+jobs:
   fast_finish: true
   include:
     - php: 7.1
       env:
         - DB=PGSQL
+        - REQUIRE_INSTALLER="$REQUIRE_RECIPE"
         - PHPCS_TEST=1
-        - PHPUNIT_TEST=framework
-        - COMPOSER_INSTALL_ARG="--prefer-lowest"
```
Comment (Cheddam) [inline .travis.yml:30]: Why remove the `--prefer-lowest` build? I guess we're not exposing one in the shared matrix?

File: .travis.yml [hunk 2]
```diff
@@ -1,106 +1,62 @@
...
-
-cache:
-  directories:
-    - $HOME/.composer/cache/files
+version: ~> 1.0
 
 addons:
   apt:
     packages:
       - tidy
```
Comment (Cheddam) [inline .travis.yml:6]: Is this still necessary? Could it be added to the shared config?

**Review by Cheddam — COMMENTED** [review]
Mostly looking good, just a few questions.

File: .travis.yml (hunk 1 above)
Comment (emteknetnz) [reply .travis.yml:30]: Yeah phase 2 thing, get --prefer-lowest into the shared matrix.  To many broken things to fix right now.

File: .travis.yml (hunk 2 above)
Comment (emteknetnz) [reply .travis.yml:6]: I'm not totally sure.  I'm pretty sure it's necessary for _something_, can't remember what though

Comment (Cheddam) [reply .travis.yml:6]: I'm sure I've seen it required in multiple modules in the past, so perhaps just shift it to shared config and drop it here.

Comment (emteknetnz) [reply .travis.yml:6]: ok

File: .travis.yml (hunk 1 above)
Comment (Cheddam) [reply .travis.yml:30]: Can we restore it and document it as a temporary deviation? Keen to keep this coverage present here until we get capacity to add it to the shared matrix.

Comment (emteknetnz) [reply .travis.yml:30]: I just added it back in

File: .travis.yml [hunk 3]
```diff
@@ -1,106 +1,62 @@
...
-  - if [[ $PHPUNIT_COVERAGE_TEST ]]; then phpdbg -qrr vendor/bin/phpunit --testsuite $PHPUNIT_COVERAGE_TEST --coverage-clover=coverage.xml; fi
-  - if [[ $PHPCS_TEST ]]; then composer run-script lint; fi
-  - if [[ $PHPCS_TEST ]]; then composer run-script php-peg; fi
-  - if [[ $PHPCS_TEST ]]; then git diff-files --quiet -w --relative=src; fi
-  - if [[ $PHPCS_TEST ]]; then git diff -w --no-color --relative=src; fi
-
-after_success:
-  - if [[ $PHPUNIT_COVERAGE_TEST ]]; then bash <(curl -s https://codecov.io/bash) -f coverage.xml; fi
+  allow_failures:
+    - php: nightly
```
[the remark this answers is not in the document]
Comment (emteknetnz) [reply .travis.yml]: Reverted

**Pull request merged**